.github/pull_request_template.md

```diff
@@ -2,6 +2,10 @@
 
 Include screenshots, photos or links if necessary.
 
+> Please update this link with the PR number and story path.
```

Comment:
I'm not sure how I feel about this change. Not all pull requests have changes related to storybook and there is always a comment from the Github bot with the PR preview link. I'd prefer to remove this change (especially as part of this work), but happy for you to push back.

Reply:
Fair point. I think the PR previews are great, and that it's important that people post direct links to a story, if it's relevant.
We could add here that the PR creator could remove this line if necessary. what do you think?